crosslink.proto

```diff
@@ -166,21 +166,22 @@ message LookupInstrumentIdRequest {
   string name = 1;
 }
 
-
 message LookupInstrumentIdResponse {
-  oneof response {
-      int32 instrument_id = 1;
-      string error = 2;
+  int64 instrument_id = 1;
+  message LookupInstrumentIdError {
+    string error = 1;
   }
+  LookupInstrumentIdError error = 2;
 }
 
 message LookupLocationIdRequest {
   string name = 1;
 }
 
 message LookupLocationIdResponse {
-    oneof response {
-        int32 location_id = 1;
-        string error = 2;
-    }
-}
\ No newline at end of file
+  int64 location_id = 1;
+  message LookupLocationIdError {
+    string error = 1;
+  }
+  LookupLocationIdError error = 2;
```

Comment:
I agree with wrapping the error behind a more general type. But will we ever return both a location and an error? If not it seems that wrapping with oneof makes the API clearer.